tuxemon/states/control/__init__.py

```diff
@@ -161,11 +161,11 @@ def change_state(
             (display_buttons[buttons.BACK], None)
         )
 
-        for key, input in key_items_map:
+        for key, button in key_items_map:
             label = f"{T.translate(key).upper()}"
             image = self.shadow_text(label)
-            item = MenuItem(image, label, None, change_state("SetKeyState", input=input))
-            item.enabled = item is not None
+            item = MenuItem(image, label, None, change_state("SetKeyState", button=button))
```

Comment:
should be `input=button`, unless you want to refactor the SetKeyState

Reply:
Yeah, I already did that in line 65. Is there an issue if I refactor it?

Reply:
i see that now!  no problem then